scripts/mobile-flow.js

```diff
@@ -32,30 +32,31 @@ module.exports = function mobileFlow(runner, argv, clientId) {
       // Then do a syncRecords for each datasets (no clientRecs yet)
         .then(() => Promise.all(datasets.map(doSyncRecords)))
       // Then do another sync of each dataset
-        .then(syncRecordsResults => Promise.all([
+        .then(initialSyncRecordsResponses => Promise.all([
           Promise.all(datasets.map(ds => doSync(`${baseUrl}/mbaas/sync/${ds}`, makeSyncBody(ds, clientId)))),
-          Promise.resolve(syncRecordsResults)
+          Promise.resolve(initialSyncRecordsResponses)
         ]))
       // Then do another syncRecords, this time with clientRecs
-        .spread((syncResults, syncRecordsResults) => Promise.all([
+        .spread((syncResults, initialSyncRecordsResponses) => Promise.all([
+          // TODO: in the browser, I see a hash in the response from syncRecords, but not here in the script
```

Comment:
@david-martin this TODO is something that could be investigated separately to the improvements to the scripts that's being done here.